.eslintrc.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
     'import/extensions': 'off',
     'mocha/no-hooks-for-single-case': 'off',
     'no-param-reassign': 'off',
+    'max-len': 'off',
```

Comment:
We shouldn't turn it off globally, most of the time code can be formatted and in other situations you can disable the rule just for one line or block of code.